cmd/influx_stress/influx_stress.go

```diff
@@ -3,125 +3,55 @@ package main
 import (
 	"flag"
 	"fmt"
-	"runtime"
-	"sort"
-	"time"
+	"os"
+	"runtime/pprof"
 
 	"github.com/influxdb/influxdb/stress"
 )
 
 var (
-	batchSize     = flag.Int("batchsize", 0, "number of points per batch")
-	concurrency   = flag.Int("concurrency", 0, "number of simultaneous writes to run")
-	batchInterval = flag.Duration("batchinterval", 0*time.Second, "duration between batches")
-	database      = flag.String("database", "", "name of database")
-	address       = flag.String("addr", "", "IP address and port of database (e.g., localhost:8086)")
-	precision     = flag.String("precision", "", "The precision that points in the database will be with")
-	test          = flag.String("test", "", "The stress test file")
+	//database  = flag.String("database", "", "name of database")
+	//address   = flag.String("addr", "", "IP address and port of database (e.g., localhost:8086)")
+
+	config     = flag.String("config", "", "The stress test file")
+	cpuprofile = flag.String("cpuprofile", "", "File where cpu profile will be written")
```

Comment:
It's late, disregard the earlier comment. I see the `cpuprofile` is omitted when it's left empty.

You may want to use Go's back tick option naming style, e.g. "Write the cpu profile to `filename`". Then the output will use `filename` as the option parameter's name in the usage text.

```
  -cpuprofile filename
    	Write the cpu profile to filename
```